capabilities.json

```diff
@@ -132,15 +132,28 @@
         "label": {
             "displayName": "Center Label",
             "properties": {
-                "fill": {
-                    "displayName": "Fill",
+                "show": {
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "color": {
```

Comment:
@ignatvilesov  I didn't find any usages of object "label" in code. Or do I make a mistake?